extensions/ftp/pom.xml

```diff
@@ -29,11 +29,38 @@
             <groupId>commons-net</groupId>
             <artifactId>commons-net</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
Ok, sure. Thanks!

On Thu, Jun 2, 2016 at 3:10 PM, Mariano Gonzalez <notifications@github.com>
wrote:

> In extensions/ftp/pom.xml
> <https://github.com/mulesoft/mule/pull/2743#discussion_r65590038>:
>
> > @@ -29,11 +29,38 @@
> >              <groupId>commons-net</groupId>
> >              <artifactId>commons-net</artifactId>
> >          </dependency>
> > +
> > +        <dependency>
>
> check with Kraan, but I'm under the impression that the java and javax
> ones are going to be runtime only
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mulesoft/mule/pull/2743/files/c98fb624efa9b9914f2b46038515d8dd47bd2159#r65590038>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG96KaETnm2Ugrj-QbnCnywu3rA6ULgeks5qHxyzgaJpZM4Isq0Y>
> .
>



-- 
<http://www.mulesoft.com/>
Guillermo Fernandes, Senior Software Engineer

Av. Corrientes 316 Entre Piso, Buenos Aires, C1043AAQ, Argentina


<https://www.linkedin.com/company/www.mulesoft.com>
<https://twitter.com/mulesoft>   <https://www.facebook.com/MuleSoft>
<https://www.youtube.com/user/mulesoftvids> We're hiring!
<http://www.mulesoft.com/careers>